clients/admin-ui/src/features/plus/plus.slice.ts

```diff
@@ -162,6 +162,8 @@ export const plusApi = createApi({
         params: { show_values },
       }),
       providesTags: ["AllowList"],
+      transformResponse: (allowList: AllowList[]) =>
+        allowList.sort((a, b) => a.name!.localeCompare(b.name!)),
```

Comment:
The name shouldn't be undefined, but just in case I'd use `(a.name ?? "").localeCompare`